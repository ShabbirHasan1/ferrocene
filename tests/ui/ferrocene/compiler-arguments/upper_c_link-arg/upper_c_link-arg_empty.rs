// check-fail
// compile-flags: -C link-arg

fn main() {}

// ferrocene-annotations: um_rustc_C_link_arg
