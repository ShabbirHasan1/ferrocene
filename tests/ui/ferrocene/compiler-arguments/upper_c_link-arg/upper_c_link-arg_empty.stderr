error: codegen option `link-arg` requires a string (C link-arg=<value>)

